zigpy/zcl/clusters/general.py

```diff
@@ -1274,10 +1277,160 @@ class PollControl(Cluster):
+        0x35 : ("Move Up (with On/Off)",(t.Optional(t.uint8_t),)),
+        0x36 : ("Move Down (with On/Off)",(t.Optional(t.uint8_t),)),
+        0x37 : ("Step Up (with On/Off)",(t.uint8_t,t.Optional(t.uint16_t))),
+        0x38 : ("Step Down (with On/Off)",(t.uint8_t,t.Optional(t.uint16_t))),
+        0x40 : ("Move Hue Stop",()),
+        0x41 : ("Move Hue Up Color",(t.Optional(t.uint8_t),)),
+        0x42 : ("Move Hue Down Color",(t.Optional(t.uint8_t),)),
+        0x43 : ("Step Hue Up Color",(t.uint8_t,t.Optional(t.uint16_t))),
+        0x44 : ("Step Hue Down Color",(t.uint8_t,t.Optional(t.uint16_t))),
+        0x46 : ("Move Saturation Up",(t.Optional(t.uint8_t),)),
+        0x47 : ("Move Saturation Down",(t.Optional(t.uint8_t),)),
+        0x48 : ("Step Saturation Up",(t.Optional(t.uint8_t),)),
+        0x49 : ("Step Saturation Down",(t.Optional(t.uint8_t),)),
+        0x4A : ("Move Color",(t.uint16_t,t.uint16_t)),
+        0x4B : ("Step Color",(t.uint16_t,t.uint16_t,t.Optional(t.uint16_t))),
+        0x45 : ("Move Saturation Stop",()),
+        0x50 : ("Lock Door",()),
+        0x51 : ("Unlock Door",()),
+        0x60 : ("Press 1 of 1",()),
+        0x61 : ("Release 1 of 1",()),
+        0x62 : ("Press 1 of 2",()),
+        0x63 : ("Release 1 of 2",()),
+        0x64 : ("Press 2 of 2",()),
+        0x65 : ("Release 2 of 2",()),
+        0x66 : ("Short press 1 of 1",()),
+        0x67 : ("Short press 1 of 2",()),
+        0x68 : ("Short press 2 of 2",()),
+    }
+
+    zigBeeLinkKey=t.KeyData([0x5A, 0x69, 0x67, 0x42, 0x65, 0x65, 0x41, 0x6C, 0x6C, 0x69, 0x61, 0x6E, 0x63, 0x65, 0x30, 0x39])
+
+    def handle_message(self,data):
+        if data[0][0] == 12 and data[0][5] == 224 :
+            src_id = int.from_bytes(data[0][1:5],byteorder='little')
+            command = data[0][5]
+            type = data[0][6]
+            opt = int.from_bytes(data[0][7:9],byteorder='little')
+            securityKey = data[0][9:25]
+            securityKeyValidation = int.from_bytes(data[0][25:29],byteorder='little')
+            counter=int.from_bytes(data[0][29:33],byteorder='little')
+            LOGGER.debug('GreenPower commissioning frame src_id : 0x%08X, command : 0x%02X, type : 0x%02X, opt : 0x%02X, counter : 0x%04X, securityKey : %s',src_id,command,type,opt,counter,securityKey)
+            payload = (
+                0x00e548,
+                src_id,
+                0x0b84,
+                type,
+                counter,
+                GreenPowerProxy.encryptSecurityKey(src_id, securityKey),
+            )
+            LOGGER.debug('payload : %s',payload)
+            tsn = self.endpoint.device.application.get_sequence()
+            hdr = foundation.ZCLHeader.cluster(tsn,1) # Pairing
+            hdr.frame_control.disable_default_response=True
+            data = hdr.serialize() + t.serialize(payload, (t.bitmap24,t.uint32_t,t.uint16_t,t.enum8,t.uint32_t,t.Struct))
+            asyncio.ensure_future(self.endpoint.device.application.broadcast(
```

Comment:
When scheduling a coro as a response to incoming message, use `self.create_catching_task` like in `Time` cluster https://github.com/zigpy/zigpy/blob/e83733847e848c9a52ef76e4a3026add640c72ec/zigpy/zcl/clusters/general.py#L529